src/main/java/com/blackducksoftware/integration/hub/alert/datasource/entity/SlackConfigEntity.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (C) 2017 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.hub.alert.datasource.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "slack_config", schema = "configuration")
+public class SlackConfigEntity extends DatabaseEntity {
+    private static final long serialVersionUID = 4872590789715928839L;
+
+    @Column(name = "channel_name")
+    private String channelName;
```

Comment:
Is this the name that will be applied to the bot? If so, I think it should always just be Hub Alert (using the constant). Are there other basic fields that would be useful to configure other than these?

Reply:
channel_name would be the name of the chat room in slack you want to send the notification to. Slack uses the name channel for this, like we do.

Reply:
Talked to Paulo and we believe we should leave the bot name up to the user but default to hub_alert when necessary